spec/models/user_spec.rb

```diff
@@ -29,4 +29,13 @@
       expect(FactoryGirl.create(:user)).to be_valid
     end
   end
+
+  describe 'Restaurant Owner role' do
+    params = {email: 'xavy@ibm.com', password: '12345678', password_confirmation: '12345678', restaurant_owner: true  }
+    user = FactoryGirl.create(:user, params)
+    let (:current_user) {user}
```

Comment:
Also, In your specs you should separate the way you name your objects from implementation. What I mean is that `current_user` as a variable name is tightly coupled to the way you USE th User model. But it's only one of the many use cases wher an object of the User class will exist. In RSpec you have access to `subject` right? I would use that instead of creating a named object.

Something like this should work

```ruby
subject { FactoryGirl.create(:user, restaurant_owner: true }
```
Does that make sense?